File: APIREST-MONGODB-CRM-PI/interfaces/proyecto.ts
import { Document, Schema } from "mongoose";
import IUsuario from "./usuario";
import ITarea from "./tarea";

export default interface IProyecto extends Document {
  nombre: Schema.Types.String;
  subtitulo: Schema.Types.String;
  descripcion: Schema.Types.String;
  estado: Schema.Types.Boolean;
  tareas: [ITarea];
  integrantes: [IUsuario];

}

File: vistamovilpi/app/src/main/java/com/example/vista_movil_pi/modelo/Proyecto.kt
package com.example.vista_movil_pi.modelo

data class Proyecto (
    val nombre: String,
    val subtitulo: String,
    val descripcion: String,
    val estado: Boolean,
    val tareas: List<Tarea>,
    val integrantes: List<Cliente>
) {
    val _id: String = ""
}